admin/client/src/components/ListGroup/ListGroup.scss

```diff
@@ -56,27 +61,23 @@
   opacity: .6;
 
   &:hover {
-		opacity: 1;
+    opacity: 1;
+
+    .list-group-item__status {
+      opacity: 1;
+    }
 	}
+
+  .list-group-item__status {
+    opacity: 0;
+  }
 }
 
 .list-group-item__status {
   display: inline-block;
   transition: opacity .2s ease-in-out;
 }
 
-.list-group-item.active .list-group-item__status {
-	color: #FFF;
-	opacity: 1;
-}
-
-.list-group-item--inactive .list-group-item__status {
```

Comment:
Sorry I meant to take a look today, it could have been an oversite from me to remove. Will take a look first thing tomorrow.